Client.xcodeproj/project.pbxproj

```diff
@@ -6693,6 +6698,7 @@
 				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
 				ALWAYS_SEARCH_USER_PATHS = NO;
 				APPLICATION_EXTENSION_API_ONLY = NO;
+				APP_IDENTIFIER_PREFIX = 9G8J6YA743;
```

Comment:
Hard to see what section this is. Is the `9G8J6YA743 ` ok here? Maybe this is for our Enterprise account? Same below.